File: Ruby/P3/Irrgarten/mainP3.rb

require_relative 'controller'
require_relative 'textUI'
require_relative 'game'
require_relative 'dice'

module Irrgarten
    class MainP3
        def self.main ()

            numero_jugadores = 2

            vista = UI::TextUI.new
            game = Game.new(numero_jugadores)
            controller = Control::Controller.new(game, vista)
            controller.play()

        end
    end
end

Irrgarten::MainP3.main

File: Java/P3/irrgaten/src/main/mainP3.java

package main;

import irrgarten.Controller.Controller;
import irrgarten.Game;
import irrgarten.UI.TextUI;

/**
 *
 * @author Omnia Infosys
 */
public class mainP3 {

    public static void main (String []args){
        
        int njugadores = 2;
        
        TextUI vista = new TextUI();
        Game game = new Game(njugadores);
        Controller controller = new Controller (game, vista);
        controller.play();
    
    }
}
